REFERENCE.md

```diff
@@ -2753,6 +2753,11 @@ Struct[{
     Optional['ProtectHome']               => Variant[Boolean, Enum['read-only', 'tmpfs']],
     Optional['BindPaths']                 => Variant[Stdlib::Unixpath,Pattern[/-\/.+/], Array[Variant[Stdlib::Unixpath,Pattern[/-\/.+/]],1]],
     Optional['BindReadOnlyPaths']         => Variant[Stdlib::Unixpath,Pattern[/-\/.+/], Array[Variant[Stdlib::Unixpath,Pattern[/-\/.+/]],1]],
+    Optional['ReadWritePaths']            => Variant[Stdlib::Unixpath,Pattern[/-\/.+/], Array[Variant[Stdlib::Unixpath,Pattern[/-\/.+/]],1]],
```

Comment:
```suggestion
    Optional['ReadWritePaths']            => Variant[Stdlib::Unixpath,Pattern[/^-\/.+/], Array[Variant[Stdlib::Unixpath,Pattern[/^-\/.+/]],1]],
```

and similar. The `-` is always at the start.

Reply:
Thanks! 
I updated the regular expression to accommodate this (from the systemd docs): 
```
Paths in ReadWritePaths=, ReadOnlyPaths=, InaccessiblePaths=, ExecPaths= and NoExecPaths= may be prefixed with "-", in which case they will be ignored when they do not exist. If prefixed with "+" the paths are taken relative to the root directory of the unit, as configured with RootDirectory=/RootImage=, instead of relative to the root directory of the host (see above). When combining "-" and "+" on the same path make sure to specify "-" first, and "+" second.
```

Tested thus: https://rubular.com/r/oATb23JSEuE0MX